src/Compiler/Driver/CompilerConfig.fs

```diff
@@ -777,6 +786,10 @@ type TcConfigBuilder =
             xmlDocInfoLoader = None
             exiter = QuitProcessExiter
             parallelReferenceResolution = ParallelReferenceResolution.Off
+            typeCheckingConfig =
+                {
+                    TypeCheckingConfig.Mode = TypeCheckingMode.Sequential
```

Comment:
Can you please make the default based on an environment variable for always-on experimental dogfooding?
I am confident our highest chance of spotting issues is by using it for everything as soon as possible (and not assuming people will pass custom switches)

Reply:
Done.